accounts/models.py

```diff
@@ -1,15 +1,39 @@
-from django.contrib.auth.models import AbstractUser
+
+from django.contrib.auth.models import AbstractUser,PermissionsMixin
 from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
 
 
 #デフォルトで用意されているUserではない
-class User(AbstractUser):
+class User(AbstractUser, PermissionsMixin):

```

Comment:
```suggestion
class User(AbstractUser):

```
ここ見逃してました。AbstractUserはPermissionMixinを継承しているのでここで再び継承する必要ないです。